lib/install/validate/validator.go

```diff
@@ -548,6 +548,9 @@ func (v *Validator) checkDatastoresAreWriteable(ctx context.Context, conf *confi
 	validStores := make(map[types.ManagedObjectReference]*object.Datastore)
 	// remove any found datastores from requested datastores
 	for _, cds := range clusterDatastores {
+		var ds mo.Datastore
```

Comment:
This shouldn't be here, accidently got included when I was talking to Mike Hagen.